tests/test_declarations/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -113,6 +117,12 @@ describe(\`does something really long and complicated so I have to write a very
   });
 });
 
+describe("does something really long and complicated so I have to write a very long name for the describe block", willResolve(
+  () => {
```

Comment:
It's not perfect. It solves keeping the string on the same line as the function name, which is the biggest visual nit, but fumbles after the comma.